### ISSUE TITLE
AVAE with convolutional architecture

### ISSUE THREAD

**Issue by amruz** [deepmind/deepmind-research#336]
I was trying to replicate the results in the paper for AVAE trained with convolutional architecture, however the training results are reallz noisy for ColorMNIST, I meant the reconstructions from the decoder and new samples generated are really bad. I just replaced the current fully connected architecture network of encoder and decoder in the repository with convolutional architecture. Are there any particular things to take care of when the architecture is changed in the current training pipeline, in terms of loss implementation for example. Are there any plans in the near future to open source the convolutional based architecture training code as well or upload any model checkpoints? Any pointers here would be really helpful. Thanks in advance!

**Comment by flyleeee** [comment]
It seems they have some troubles in opening source code😅
